### PR TITLE
add flyers dir and cheatsheet French translation

### PR DESCRIPTION
**Pull request by sogalgeeko**
I started to translate this nice flyer (which is very appreciated by the public on booth and FOSS events) quite some time ago from and... ukranian I guess, translation. It was the only SVG source I could find.
If no English source file can be found, I will create one.

### CODE REVIEW

File: flyers/fr/openSUSE-Cheat-Sheet-BG_fr.svg [hunk 1]
```diff
@@ -0,0 +1,5589 @@
...
+     style="fill:#66bf2e;fill-opacity:1;fill-rule:nonzero;stroke:none" />
+  <path
+     inkscape:connector-curvature="0"
+     id="path235"
+     d="m 1191.7227,128.83594 0,-0.82032 -0.02,0 c -0.1797,0.32032 -0.4101,0.55469 -0.6914,0.70313 -0.2773,0.15234 -0.6094,0.22656 -0.9883,0.22656 -0.3398,0 -0.625,-0.0469 -0.8515,-0.13672 -0.2266,-0.0898 -0.4102,-0.21875 -0.5508,-0.38281 -0.1367,-0.16797 -0.2383,-0.36719 -0.293,-0.59765 -0.059,-0.22657 -0.086,-0.48438 -0.086,-0.76172 l 0,-3.40235 0.8516,0 0,3.5 c 0,0.32032 0.09,0.57422 0.2773,0.76172 0.1875,0.18359 0.4453,0.27734 0.7695,0.27734 0.2617,0 0.4883,-0.0391 0.6758,-0.11718 0.1914,-0.082 0.3477,-0.19531 0.4766,-0.33985 0.125,-0.14843 0.2226,-0.32031 0.2851,-0.51562 0.062,-0.19922 0.094,-0.41406 0.094,-0.64453 l 0,-2.92188 0.8515,0 0,5.17188 z m 0,0"
+     style="fill:#66bf2e;fill-opacity:1;fill-rule:nonzero;stroke:none" />
+  <path
+     inkscape:connector-curvature="0"
+     id="path237"
+     d="m 1194.6016,123.66406 0,0.82032 0.02,0 c 0.3555,-0.625 0.9141,-0.94141 1.6797,-0.94141 0.3398,0 0
```
Comment (hellcp) [inline flyers/fr/openSUSE-Cheat-Sheet-BG_fr.svg]: YaST no longer has GTK UI option (since a few years back)

Comment (hellcp) [inline flyers/fr/openSUSE-Cheat-Sheet-BG_fr.svg]: For some reason osc commands are the only ones that lack a space after `#`

Comment (hellcp) [reply flyers/fr/openSUSE-Cheat-Sheet-BG_fr.svg]: Not to mention Open Build Service is written as 3 words, not one word

Comment (hellcp) [inline flyers/fr/openSUSE-Cheat-Sheet-BG_fr.svg]: Ah, one other place where there is no space after `#`

Comment (sogalgeeko) [reply flyers/fr/openSUSE-Cheat-Sheet-BG_fr.svg]: Fixed, I got misleaded by the fact that the option "--gtk" still exists in `yast --help` but, indeed, falls back to "--qt".

**Comment by sogalgeeko** [comment]
Should be all good by now. Is there any English source files somewhere ? If not, is it worth that I translate those flyers in English (i.e. do they will be printed again for FOSS events ?)

**Comment by hellcp** [comment]
I certainly don't know where those would be, maybe @ddemaio does?

**Comment by sogalgeeko** [comment]
In https://github.com/openSUSE/artwork/pull/37/commits/fa2d618a1c573450186c795a3e4e5dcf7ca58588 and https://github.com/openSUSE/artwork/pull/37/commits/406a95471be29111613ab4e30517b6fbee00b52a I recreated and cleaned the source SVG in English, so now we have the source.

**Pull request merged**